numpy/lib/function_base.py

```diff
@@ -1975,28 +1984,28 @@ def corrcoef(x, y=None, rowvar=1, bias=0, ddof=None):
         variable, with observations in the columns. Otherwise, the relationship
         is transposed: each column represents a variable, while the rows
         contain observations.
-    bias : int, optional
-        Default normalization is by ``(N - 1)``, where ``N`` is the number of
-        observations (unbiased estimate). If `bias` is 1, then
-        normalization is by ``N``. These values can be overridden by using
-        the keyword ``ddof`` in numpy versions >= 1.5.
-    ddof : int, optional
-        .. versionadded:: 1.5
-        If not ``None`` normalization is by ``(N - ddof)``, where ``N`` is
-        the number of observations; this overrides the value implied by
-        ``bias``. The default value is ``None``.
 
     Returns
     -------
-    out : ndarray
+    R : ndarray
         The correlation coefficient matrix of the variables.
 
     See Also
     --------
     cov : Covariance matrix
 
-    """
-    c = cov(x, y, rowvar, bias, ddof)
+    Notes
+    -----
+    This function accepts but discards arguments `bias` and `ddof`.  This is
+    for backwards compatibility with previous versions of this function.  These
+    arguments had no effect on the return values of the function and can be
+    safely ignored in this and previous versions of numpy.
+    """
+    if len(args) or kwargs.pop('bias', _DefaultArg) is not _DefaultArg:
```

Comment:
We are opening the door to lots of nonsense input going through silently, e.g. extra keyword or positional arguments, `bias` and `ddof` being provided as both positional and keyword simultaneously... May not be worth adding all those checks for something we actually want to deprecate, but...